sources/core/Stride.Core.Tests/TestProfiler.cs

```diff
@@ -132,7 +132,7 @@ public void TestWitAttributes()
         }
 
 
-        private static Regex matchElapsed = new Regex(@"Elapsed = ([\d\.]+)");
+        private static Regex matchElapsed = new Regex(@"Elapsed = ([\d.,]+)");
```

Comment:
Is that comma (or period) culture-specific? If it is, the tests will fail for other locales, isn't it?

Also, in the function above there's a typo I think. It should be `TestWithAttrobutes` instead of `TestWitAttributes`?

Reply:
> Is that comma (or period) culture-specific? If it is, the tests will fail for other locales, isn't it?

It was . (and \\) failing in "comma culture", now both work, are there (realistically) cultures that don't use . or , as decimal separator? I am tempted to rewrite these tests to not rely on string formatting and parsing, when I add new tests.

> Also, in the function above there's a typo I think. It should be `TestWithAttrobutes` instead of `TestWitAttributes`?

Noted, but I won't add the new typo 🤣